doc/version_history.rst

```diff
@@ -6,6 +6,16 @@
 Version History
 ###############
 
+v1.1.3
+------
+
+* Add the new commands to set the emergency acceleration and jerk in the run time.
```

Comment:
`in the run time` -> `at runtime`

Reply:
Updated in the commit: [3ed04af](https://github.com/lsst-ts/ts_mtrotator/pull/86/commits/3ed04af203aa5ccebcfea3464dfa1b4652aea1f6). Thanks!